README.md

```diff
@@ -1 +1,52 @@
 # adobeio-samples-custom-events-using-journaling
```

Comment:
BTW is it for custom events or any events? maybe a more generic name `adobeio-samples-events-handling-journaling`?

Reply:
hmm.. good idea, will change it